ResearchKit/Common/ORKTextFieldView.m

```diff
@@ -185,7 +184,7 @@ - (void)ork_setSuffix:(NSString *)suffix withColor:(UIColor *)color {
     if (suffix.length == 0) {
         return;
     }
-    _suffixLabel = [self ork_createTextLabelWithTextColor:color ?: [UIColor grayColor]];
+    _suffixLabel = [self ork_createTextLabelWithTextColor:(color ?: [UIColor ork_midGrayTintColor])];
```

Comment:
This color is sometimes used for the *unit suffix label* when editing an `ORKTextFieldView` with no placeholder. You can see this if you revert this change, tap on the first text field of the first step on *Mini Form*, tap on the second field, and then tap on the first field again (the unit suffix label turns darker).

`ork_midGrayTintColor` is used as the color in all other `placeholder`s and *unit suffix* instances, so I figured it should be used here too.

But maybe we should also review the actual color-chosing logic, because it's weird that this color only gets used the second time you tap on the field but not the first one.